client/signup/controller.js

```diff
@@ -246,6 +248,45 @@ export default {
 			context.store.dispatch( setSelectedSiteId( null ) );
 		}
 
+		if ( flowName === 'onboarding' || flowName === 'onboarding-plan-first' ) {
+			context.primary = (
+				<Experiment name="domain_step_plan_step_swap">
```

Comment:
```suggestion
				<Experiment name="signup_domain_plan_step_swap">
```
I think the experiment name needs to be updated!